index.js

```diff
@@ -21,7 +23,55 @@ mongoose
   })
   .then(() => {
     // Run your code here, after you have insured that the connection was made
+   /* THIS IS ITERATION 2!!!!!!!!!
+    Recipe.create( data[0] )
+      .then((recipe) => {
+        console.log(recipe.title)
+
+      })
+      .catch(()=> {
+        console.log()
+      })
+      */
+
+     Recipe.insertMany(data)
+        .then((recipes) => {
+          //console.log(recipes)
+          for(let recipe in recipes){
+            console.log(recipes[recipe].title)
+          }
+
+          let updatedPromise = Recipe.updateOne({title: "Rigatoni alla Genovese"}, {$set: {duration: 100}})
```

Comment:
for mongoose methods that update the data. you don't need to use $set, as it already knows the second argument that you pass will be the data to be updated.